plugins/org.jkiss.dbeaver.tasks.ui.view/src/org/jkiss/dbeaver/tasks/ui/view/DatabaseTasksTree.java

```diff
@@ -542,6 +553,7 @@ private void addCheckedItem(TreeItem item, List<DBTTask> tasks) {
                     thisFolderTasks = allTasks.stream().filter(task -> task.getTaskFolder() == taskFolder).collect(Collectors.toList());
                     // folderProject will be null in this case. It's ok
                 }
+                thisFolderTasks.sort((o1, o2) -> o1.getName().compareToIgnoreCase(o2.getName()));
```

Comment:
```suggestion
                thisFolderTasks.sort(DBUtils.nameComparatorIgnoreCase());
```

Reply:
Is sorting here essential? We can define a custom comparator for the viewer instead.